### ISSUE TITLE
When passing in Meteor.user() through another HOC I have a weird issue

### ISSUE THREAD

**Issue by clayne11** [kadirahq/react-komposer#25]
I have a component that has two composeWithTracker HOCs components chained together.

One of them simply fetches the Meteor.user() and returns the props {user}. The second container users the userId to fetch data out of a Meteor collection.

If I have a tab open with this container, then open a new tab to my application, when that tab loads, it clears the data on my original tab. My subscription in the second HOC hits "not ready" and never becomes ready.

The issue only occurs when the Meteor.user() HOC is chained in front of the other HOC. When I put the Meteor.user() HOC after the other one, everything works as expected.

**Comment by clayne11** [comment]
I solved it on my own.

**Comment by arunoda** [comment]
Could you document it here and test us how you solved it?

**Comment by clayne11** [comment]
More info on this:

Using Meteor 1.2 and browserify.

When I had composeWithTracker nested under own HOC that used ReactMeteorData the subscriptions would sometimes get stopped. It would consistently happen when I opened a new browser tab going to the site.

Interestingly enough, when I replaced the ReactMeteorData HOC with another composeWithTracker, it all worked fine. For some reason, though, they didn't play nicely together.

I ended up moving off of react-komposer because I was having issues with some of my integration tests since Tracker was not always available in the global scope and since react-komposer doesn't directly depend on Tracker.
